engines/bops_config/app/controllers/bops_config/application_types/document_tags_controller.rb

```diff
@@ -43,7 +43,11 @@ def set_document_tags
       end
 
       def next_path
-        application_type_path(@application_type)
+        if @application_type.decision_set?
```

Comment:
Could we use `configured?` instead here.
And if the default is `{}` this will always return true right